.github/workflows/test-build.yml

```diff
@@ -24,24 +24,34 @@ jobs:
             local branch=$1
             local found=0
             shift
-            for endpoint in $@; do
-              local json=$(curl -s $endpoint)
-              local branch_found=$(echo $json | jq -c ".[] | select(.name==\"$branch\") | .name")
-              # to track what we found
-              >&2 echo "$endpoint -> $branch_found"
-              if [[ $branch_found != "" ]]; then
-                found=$((found += 1))
-              fi
-            done
+
+            local dockerhub_endpoint=$1
+            local dockerhub_json=$(curl -s $dockerhub_endpoint)
+            local dockerhub_branch_found=$(echo $dockerhub_json | jq -c ".results[] | select(.name==\"$branch\") | .name")
```

Comment:
Sorry this change is tacked onto this - wanted to make sure this PR worked while DockerHub was in brownouts..

Dockerhub's v1 API [is deprecated and will be switched off on 5th September](https://www.docker.com/blog/docker-hub-v1-api-deprecation/). They're currently doing [brownouts](https://twitter.com/Docker/status/1561707850543632385).